src/categories/queries.ts

```diff
@@ -8,6 +8,13 @@ import {
 } from "./types/CategoryDetails";
 import { RootCategories } from "./types/RootCategories";
 
+export const fragmentMoney = gql`
```

Comment:
`src/products/queries.ts:48` here's the same fragment

Reply:
fixed